src/service/rpc/other/AbstractStellarRpc.ts

```diff
@@ -0,0 +1,337 @@
+    const { offerId, ...rest } = params
+    return this.sendGet({ path: `/offers/${offerId}/trades`, queryParams: rest })
+  }
+
+  getOrderBook(params: GetOrderBookParams): Promise<Order> {
+    const { sellingAssetType, ...rest } = params
+    return this.sendGet({ path: `/order_book/${sellingAssetType}`, queryParams: rest as QueryParams })
+  }
+
+  getTradeAggregations(params: GetTradeAggregationsParams): Promise<TradeAggregation> {
+    const { baseAssetType, counterAssetType, ...rest } = params
+    return this.sendGet({
+      path: `/trade_aggregations/${baseAssetType}${counterAssetType}`,
+      queryParams: rest as QueryParams,
+    })
+  }
+
+  getTrades(params: GetTradesParams): Promise<Links & Trade> {
+    return this.sendGet({ path: '/trades', queryParams: params as QueryParams })
+  }
+
+  getTransaction(params: GetTransactionParams): Promise<Transaction> {
+    const { transactionHash, ...rest } = params
+    return this.sendGet({ path: `/transactions/${transactionHash}`, queryParams: rest })
+  }
+
+  getTransactionOperations(params: GetTransactionOperationsParams): Promise<Operation> {
+    const { transactionHash, ...rest } = params
+    return this.sendGet({ path: `/transactions/${transactionHash}/operations`, queryParams: rest })
+  }
+
+  getTransactionEffects(params: GetTransactionEffectsParams): Promise<Links & Effect> {
+    const { transactionHash, ...rest } = params
+    return this.sendGet({ path: `/transactions/${transactionHash}/effects`, queryParams: rest })
+  }
+
+  getTransactions(params: GetTransactionsParams): Promise<Links & Transaction> {
+    return this.sendGet({ path: '/transactions', queryParams: params as QueryParams })
+  }
+
+  getOperation(params: GetOperationParams): Promise<Operation> {
+    const { id, ...rest } = params
+    return this.sendGet({ path: `/operations/${id}`, queryParams: rest })
+  }
+
+  getOperationEffects(params: GetOperationEffectsParams): Promise<Links & Effect> {
+    const { id, ...rest } = params
+    return this.sendGet({ path: `/operations/${id}/effects`, queryParams: rest })
+  }
+
+  getOperations(params: GetOperationsParams): Promise<Operation> {
+    return this.sendGet({ path: '/operations', queryParams: params as QueryParams })
+  }
+
+  getPayments(params: GetPaymentsParams): Promise<Links & OperationResponse> {
+    return this.sendGet({ path: '/payments', queryParams: params as QueryParams })
+  }
+
+  getStrictReceivePaymentPaths(params: GetStrictReceivePaymentPathsParams): Promise<Path> {
+    const { destinationAssetType, destinationAmount, ...rest } = params
+    return this.sendGet({
+      path: `/paths/strict-receive${destinationAssetType}${destinationAmount}`,
+      queryParams: rest,
+    })
+  }
+
+  getStrictSendPaymentPaths(params: GetStrictSendPaymentPathsParams): Promise<Path> {
+    const { sourceAssetType, sourceAmount, ...rest } = params
+    return this.sendGet({
+      path: `/paths/strict-send${sourceAssetType}${sourceAmount}`,
```

Comment:
same here, how does that work without slashes?

Reply:
it is in their api :-D